.github/workflows/assemble-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Assemble release and generate versions file
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version number'
+        required: true
+jobs:
+  create-new-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Checkout versions
+        uses: actions/checkout@v2
+        with:
+          repository: backstage/versions
+          path: versions
+          token: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      - name: Configure Git
+        run: |
+          git config --global user.email noreply@backstage.io
+          git config --global user.name 'Github versions workflow'
+      - name: Create release
+        run: |
+          node scripts/assemble-manifest.js ${{ github.event.inputs.version }} versions/v1/releases
+          cd versions
+          if [[ "${{ github.event.inputs.version }}" == *"next"* ]]; then
```

Comment:
Perhaps a followup, but wonder if it's worth baking this into the script? I think it would be best to do a semver comparison and only write the tag if it's actually a newer release. Right now we'll end up with some strangeness if we're doing backfills

Reply:
I'm thinking backfills will be a one-off thing that we can sort however locally. But it might still make sense to update tags in the script instead? What do you think?